docs/index.md

```diff
@@ -1,54 +1,75 @@
-<pre>aws s3api get-object content.txt --profile foo --bucket bar --key baz && cat content.txt && rm content.txt</pre>
-</details>
-<details>
-<summary>Exec a command on a Kubernetes pod or GCP Compute Instance?</summary>
-<pre>kubectl exec foo uname</pre>
-<pre>gcloud compute ssh foo --command uname</pre>
-</details>
-<details>
-<summary>Find all AWS EC2 instances with a particular tag, or Docker container with a specific label?</summary>
-<pre>aws ec2 describe-instances --profile foo --query 'Reservations[].Instances[].InstanceId' --filters Name=tag-key,Values=owner --output text</pre>
-<pre>docker ps --filter “label=owner”</pre>
-</details>
-
-If not, then try clicking on the arrows to see the recommended way of doing those tasks. Does it bother you that each of those is a bespoke, cryptic incantation of various vendor-specific tools? It's a lot of commands you have to use, applications you need to install, and DSLs you have to learn just to do some pretty basic tasks. In Wash, these basic tasks are simple. You'll find that
-
-<details>
-<summary>Listing stuff is as easy as <code>ls</code></summary>
-<pre>ls aws/foo/resources/ec2/instances</pre>
-<pre>ls kubernetes/foo/bar/pods</pre>
-</details>
-<details>
-<summary>Reading stuff is as easy as <code>cat</code>'ing a file</summary>
-<pre>cat gcp/foo/compute/bar/console.out</pre>
-<pre>cat aws/foo/resources/s3/bar/baz</pre>
-</details>
-<details>
-<summary>Execing a command is as easy as <code>wexec</code></summary>
-<pre>wexec kubernetes/foo/bar/pods/baz uname</pre>
-<pre>wexec gcp/foo/compute/bar uname</pre>
-</details>
-<details>
-<summary>Finding stuff is as easy as <code>find</code></summary>
-<pre>find aws/foo -k '*ec2*instance' -meta '.tags[?].key' owner</pre>
-<pre>find docker -k '*container' -meta '.labels.owner' -exists</pre>
-</details>
-
-And this is only scratching the surface of Wash's capabilities. Check out the screencast below
-
-<script id="asciicast-MkbZKZZcmokHQ8z8jTv0PQs9F" src="https://asciinema.org/a/mX8Mwa75rr1bJePLi3OnIOkJK.js" async></script>
-
-and the [tutorials](tutorials) to learn more.
+
+<p class="intro-normal">A UNIX-like shell for managing your cloud native and non-cloud native things.</p>
+
+<a href="{{ '/getting_started' | relative_url }}" style="font-size: 22px;">
+<strong>Get started</strong>
+</a>
+
+<p class="intro-large"><strong>With Wash</strong></p>
+
+* <p class="intro-normal">You can use <code>ls</code> to list, <code>cat</code> to read, and <code>wexec</code> to run commands on all your things. No more switching between confusing CLI tools.</p>
+* <p class="intro-normal">You can use <code>find</code> to filter anything on anything. No more complicated query DSLs.</p>
+* <p class="intro-normal">You can <code>cd</code> through a vendor's API. No more navigating complex console UIs.</p>
+
+<p class="intro-large"><strong>See for yourself</strong></p>
+
+<!-- Display the demos -->
+<select id="demo">
```

Comment:
The tabulated options in https://jsfiddle.net/ejy5nv0h/ look weird on mobile devices. It's also painful to get the screencasts to load correctly. Styling it right is also difficult. Thus, I went w/ a drop-down menu instead because it looked good-enough on everything and the JS wasn't too bad.

Reply:
We should probably style this so it blends a little better.

Reply:
Updated to style the dropdown.

Reply:
I'm not sure what "looks weird" means, and getting the screencasts to show up is probably fixable. Switching your dropdown to something like https://www.w3schools.com/howto/howto_js_tabs.asp seems like it should be pretty easy too. But I can play with that if you want.

Reply:
Oh, "looks weird" means it doesn't look pretty. For example, on large screens it looks something like

```
FOO BAR BAZ QUX
```

On medium screens
```
FOO BAR
BAZ QUX
```

On smaller screens:
```
FOO
BAR
BAZ
QUX
```

(Try minimizing https://jsfiddle.net/ejy5nv0h/ to see this in effect)

The tab interface was meant to (1) show what we're demo'ing and (2) make it obvious that we're demo'ing multiple things. I think the drop-down satisfies (1) and (2), so am not going to play around with the tab-interface. Am happy to review a follow-up PR though.